overlord/devicestate/devicemgr.go

```diff
@@ -141,6 +144,17 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	return m, nil
 }
 
+// StartUp implements StateStarterUp.Startup.
+func (m *DeviceManager) StartUp() error {
+	if !release.OnClassic && m.SystemMode() == "run" {
```

Comment:
starting with run only seem the safest but we will also mount save in recover at some point, we should leave a TODO about that here